terraform/modules/route53/main.tf

```diff
@@ -0,0 +1,26 @@
+
+data "aws_route53_zone" "droits" {
```

Comment:
I'm not sure this is a good idea. 

As the hosted zone needs to exist only in prod, then the records need to be manually created for each env in there (Currently manually done). The only benefit of this would be for it to maintain itself for production, but it's not really needed. 


Leave Route53 out of TF for the time being.